source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,177 @@
+
+..warning:: After attaching the SB, double check the charger polarities with a multimeter before plugging in the first battery.
+
+Some FRC vendors sell chargers with red SB50 connectors pre-attached.
+
+Battery Evaluation Tools
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Voltmeter** or **Multimeter**
+
+A voltage reading will give you a good snapshot of what the Voc (Voltage open circuit, or "float voltage") is. Anything from 13.0 to 14.0 may show up for charged batteries, but depends as much on the individual battery as it does on real state of charge. Below 13V, it's worth trying to charge it more.
+
+**Battery Charger**
+
+If your battery charger has Maintenance Mode indicator, such as a GREEN LED, you can use that indicator to tell you whether you are READY.
+
+Some chargers will cycle between "CHARGING" and "READY" periodically. This is a "maintenance" behavior, sometimes associated with the battery cooling off after a tough match, and being able to accept more charge.
+
+**Load Tester**
+
+A battery load tester can be used as a quick way to determine the detailed readiness of a battery. It may provide information like: open-load voltage, voltage under load, internal resistance, and state of charge. These metrics can be used to quickly confirm that a battery is ready for a match and even possibly help to identify some long term problems with the battery.
+
+.. image:: images/robot-battery/beak.png
+  :alt: Output screen of a common load tester showing properties of the battery.
+
+Ideal internal resistance should be less than 0.015 Ohms. The manufacturer specification for most batteries is 0.011 Ohms. If a battery gets higher than 0.02 Ohms it is a good idea to consider not using that battery for competition matches.
+
+If a battery shows significantly lower voltages at the higher test current loads, it may not be done charging, or it may need to be retired.
+
+Understanding Battery Voltages
+------------------------------
+
+A "12V battery" is anything but 12V.
+
+Fully charged, a battery can be anywhere from 13 to 14 volts open circuit (Voc). Open circuit voltage is measured with *nothing* connected.
+
+Once a load (like a robot) is connected, and any amount of current is flowing, the battery voltage will drop. So if you check a battery with a Voltmeter, and it reads 13.5, and then connect it to your robot and power on, it will read lower, maybe 13.1 on the Driver Station display. Those numbers will vary with every battery and specific robot, see Characterization below. Once your robot starts running, it will pull more current, and the voltage will drop further.
+
+Depleted batteries will read between 11.5 and 12.5 volts on an idle robot. While batteries can be run lower, it will risk permanent battery damage.
+
+(SLA battery voltage and current performance also depends on temperature: cool batteries are happy batteries.)
```

Comment:
I don't think the ( ) are needed here. I think it can just be "Battery voltage and current...."